packages/calypso-build/typescript/ts-package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig",
+
+	"compilerOptions": {
+		// Allow composed projects
+		"composite": true,
+		"disableReferencedProjectLoad": true,
```

Comment:
> In multi-project TypeScript programs, TypeScript will load all of the available projects into memory in order to provide accurate results for editor responses which require a full knowledge graph like ‘Find All References’.
>
> If your project is large, you can use the flag disableReferencedProjectLoad to disable the automatic loading of all projects. Instead, projects are loaded dynamically as you open files through your editor.

[`disableReferencedProjectLoad`](https://www.typescriptlang.org/tsconfig#disableReferencedProjectLoad)

Cool! Have you seen a speedup?

Reply:
Based on opening a few dirs in VSCode, none. But all our packages are small, considering I'm not applying this config to `./client` yet, so probably not conclusive.